exercises/node-javascript/multi-cmk-complete/store.js

```diff
@@ -0,0 +1,85 @@
+const {
```

Comment:
@mattsb42-aws @lizroth Here is the complete store file. Both CMKs have been added.